runtime/syntax/asciidoc.vim

```diff
@@ -25,6 +25,25 @@ if exists("b:current_syntax")
   finish
 endif
 
+" Conceal Systems
+let s:conceal = ''
+let s:concealends = ''
+let s:concealcode = ''
+if has('conceal') && get(g:, 'vim_asciidoc_conceal', 1)
+  let s:conceal = ' conceal'
+  let s:concealends = ' concealends'
+endif
+if has('conceal') && get(g:, 'vim_asciidoc_conceal_code_blocks', 1)
+  let s:concealcode = ' concealends'
+endif
+
+" additions to HTML groups
+if get(g:, 'vim_asciidoc_emphasis_multiline', 1)
```

Comment:
same here

Reply:
This is parameter to use build in vim syn-oneline active on specific region to support html syntax on mark down like `<em></em>  <p></p>`.